src/Handler.cpp

```diff
@@ -461,7 +462,7 @@ namespace mediasoupclient
 		if (locaIdIt == this->mapMidTransceiver.end())
 			MSC_THROW_ERROR("associated RtpTransceiver not found");
 
-		auto* transceiver = locaIdIt->second;
+		auto transceiver = locaIdIt->second;
```

Comment:
Why is the `*` removed? It's indeed a pointer and we use `auto*` for pointers.

Reply:
Is not needed. The compiler will give you a warning if you keep it. I've gone line by line dealing with the compiler.

Reply:
AFAIK we use `auto*` in mediasoup Worker, don't we?

Reply:
Unresolving conversation to clarify it.

Reply:
> AFAIK we use auto* in mediasoup Worker, don't we?

We don't use `rtc::scoped_refptr` pointers in mediasoup.

<img width="1618" alt="Screenshot 2024-02-13 at 16 34 54" src="https://github.com/versatica/libmediasoupclient/assets/732340/35b51237-c5f9-4873-be95-f72950e75162">

Reply:
My assumption is that we want to use `auto* xxx`  if later we call methods on `xxx` using `xxx->foo()` rather than `xxx.foo()`. 

But now, the error you show breaks my assumption. So ok.